Integration/TrustedFormIntegration.php

```diff
@@ -156,9 +163,12 @@ public function doEnhancement(Lead $lead)
                                 $lead->addUpdatedField('trusted_form_share_url', $data->share_url);
                                 $persist = true;
                             }
-                            $this->logger->info(
-                                'TrustedForm: Contact '.$identifier.' '.(!$persist ? 'NOT ' : '').'updated. '.(!empty($data->message) ? $data->message : '')
-                            );
+
+                            if ($persist) {
+                                $this->logger->info(
+                                    'TrustedForm: Contact '.$identifier.' '.(!$persist ? 'NOT ' : '').'updated. '.(!empty($data->message) ? $data->message : '')
```

Comment:
Ternary operator here is redundant, $persist is already true

Reply:
True, let's fix next time we swing through this.